src/sentry/coreapi.py

```diff
@@ -369,12 +380,17 @@ def parse_client_as_sdk(self, value):
         }
 
     def should_filter(self, project, data, ip_address=None):
-        # TODO(dcramer): read filters from options such as:
-        # - ignore errors from spiders/bots
-        # - ignore errors from legacy browsers
         if ip_address and not is_valid_ip(ip_address, project):
             return True
 
+        release = data.get('release')
+        if release and not is_valid_release(release, project):
+            return True
```

Comment:
I don't know about logging but we should definitely start tracking some metrics based on what filtering criteria caused them to be discarded